src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -3056,6 +3056,51 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorModel'
+  /api/repository/v1/snapshotAccessRequests/{id}/reject:
+    post:
```

Comment:
Is this `post`? I think this is idempotent so `put` seems more appropriate to me.

Reply:
I wasn't sure what to make it. `put` works